casper/src/main/scala/coop/rchain/casper/util/comm/DeployRuntime.scala

```diff
@@ -16,9 +16,14 @@ object DeployRuntime {
   def propose[F[_]: DeployService: Monad](): F[Unit] =
     DeployService[F].createBlock().flatMap {
       case Some(block) =>
-        DeployService[F].addBlock(block)
+        println(s"Response: Successfully created block")
+        for {
+          response <- DeployService[F].addBlock(block)
+          _        = println(s"Response: ${response._2}")
```

Comment:
println leftover

Reply:
How does ._2 work here? What is the type of response?

Reply:
`response` is of type `(Boolean, String)`.

The `println` does look a little funny on the other side of an `=` like that though. I think using `Capture` like we do in `deployFileProgram` would be better.